pandas/tests/indexes/test_base.py

```diff
@@ -738,6 +742,13 @@ def test_difference(self):
         self.assertEqual(len(result), 0)
         self.assertEqual(result.name, first.name)
 
+        # mixed, GH 13432
+        idx1 = Index([0, 1, 'A', 'B'])
```

Comment:
these tests don't test very much; only simple cases for a basic ``Index``. The real tests are in ``indexes/common.py`` where ALL indexes are tested.